src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -1,5 +1,3 @@
-{# This file is partially duplicated in TwigBundle/Resources/views/base_js.html.twig. If you
-   make any change in this file, verify the same change is needed in the other file. #}
```

Comment:
I would change the reference to `src/Symfony/Component/ErrorHandler/Resources/assets/js/exception.js` instead of removing it.